docker/all-in-one/init/configure-gotrue.sh

```diff
@@ -5,6 +5,7 @@ touch /var/log/services/gotrue.log
 
 /usr/local/bin/configure-shim.sh /dist/gotrue /opt/gotrue/gotrue
 
+sed -i "s|api_external_url|${API_EXTERNAL_URL:-http://localhost}|g" /etc/gotrue.env
```

Comment:
Seems like this was recently made to be mandatory; this fallback URL won't work, but not a problem until we add a test that uses it.

(on hosted platform this would be `https://ref.supabase.co/auth/v1`)